ios/activity.h

```diff
@@ -0,0 +1,46 @@
+/******************************************************************************\
+ * Copyright (c) 2004-2020
+ *
+ * Author(s):
+ *  AronVietti
```

Comment:
Is this "activity.h" and "activity.m" a copy from the files in mac?

Reply:
Yes.

Reply:
I do not want to have exact copies. Why not using the existing files?

Reply:
We could use the Mac ones, but in my opinion this would be confusing since we include Mac files on iOS (while both are different operating systems). The cleanest way would be to create an "Apple" folder with only these files. Not even 100% sure if they are needed on iOS since the server stopped once I locked my device.

Reply:
Please check if it is really required. If not, just remove it. Otherwise you should use the one in the mac directory. It is not the cleanest way but better than having two identical files.

Reply:
Should be gone now. The behaviour of the headless server is the same without this. So I think we don't need this file. I've now added the typical app delegate code to the app delegate files, but it doesn't seem to change anything (except that the warning I mentioned has now disappeared)